projects/Mallard/src/services/remote-config.ts

```diff
@@ -1,26 +1,72 @@
 import remoteConfig from '@react-native-firebase/remote-config'
-import { errorService } from './errors'
 
 // see https://rnfirebase.io/remote-config/usage for docs
-export const initialiseRemoteConfig = async () => {
-    const activated = await remoteConfig()
-        .setDefaults({
-            apple_sign_in: false,
-            lightbox_enabled: false,
-            remote_logging_enabled: true,
-        })
-        // fetch config, cache for 5mins
-        // NOTE: this cache persists when app is reloaded
-        .then(() => remoteConfig().fetch(300))
-        // activate() replaces the default config with what has been fetched
-        .then(() => remoteConfig().activate())
-        .catch(error => errorService.captureException(error))
-
-    if (activated) {
-        console.log('Remote config defaults set, fetched & activated!')
-    } else {
-        console.log(
-            'Remote config not activated - using default or cached values.',
-        )
+
+export type RemoteStringValue = string | undefined
+
+interface RemoteConfig {
+    init(): void
+    getBoolean(key: string): boolean
+    getString(key: string): RemoteStringValue
+}
```

Comment:
What's the motivation for having this in an interface when it's not used anywhere else?

Reply:
My intention was to keep a hint for the future use case where we may use remote values that are `string` or `number`.

Reply:
👍